src/components/FlowChart/FlowSelection.jsx

```diff
@@ -1,56 +1,118 @@
-import React from "react";
+import React, { useState } from "react";
 import { Dropdown } from "react-bootstrap";
 
+function SelectionIndicator({ selectedId, itemId }) {
+  return selectedId === itemId ? (
+    <i className="material-icons text-primary">radio_button_checked</i>
+  ) : (
+    <i className="material-icons">radio_button_unchecked</i>
+  );
+}
+
+function ConnectionIndicator({ connected, show }) {
+  if (!show) return null;
+  return connected ? (
+    <i className="material-icons ml-2 text-success">wifi</i>
+  ) : (
+    <i className="material-icons ml-2 text-warning">wifi_off</i>
+  );
+}
+
+function TitleConnectionIndicator({ connected, show }) {
+  if (!show) return null;
+  return connected ? (
+    <i className="material-icons text-white ml-1 mr-2">wifi</i>
+  ) : (
+    <i className="material-icons text-warning ml-1 mr-2">wifi_off</i>
+  );
+}
+
+function DeleteFlowButton({ show, deleteFlow }) {
+  if (!show) return null;
+  return (
+    <i className="material-icons text-danger float-right" onClick={deleteFlow}>
```

Comment:
Here you call the prop `deleteFlow`.
If you see the prop it's more important to understand that it's happening onClick.
The name of the prop can only say something about how the prop is used within the component, not what it actually holds.
Example
```jsx
<DeleteFlowButton deleteFlow={createFlow} />
```
In another component you did it differently and imho in a better way
```jsx
<DeleteFlowButton onClick={createFlow} />
```
Here you see right away, that it's about passing a onClick handler without having to read the component file/code.